### ISSUE TITLE
Portable version

### ISSUE THREAD

**Issue by amacado** [yarseyah/sentinel#21]
Are there any plans on a portable version of sentinel which does not require an installation?

**Comment by A9G-Data-Droid** [comment]
I see movement towards .NET 5 where the ability to generate a single EXE becomes simple: 

https://docs.microsoft.com/en-us/dotnet/core/deploying/single-file

**Comment by yarseyah** [comment]
There is already a basic port of this to .net 5 that is now on the master branch.  As part of that I've removed the Squirrel based installer.  I've not quite got ready to release it yet.

**Comment by A9G-Data-Droid** [comment]
So I successfully built a portable binary and it came in at a whopping 85MB. I've never used .NET5 or the new single-file deployment so I'm not sure if that's normal. It's got the whole kitchen sink in one file. It does run but I don't feel like that's an acceptable file size when the normal version is 10 files that total 2.41MB in size.

**Comment by amacado** [comment]
@A9G-Data-Droid It's pretty normal you see a explosion in file size when you compile it to single-file application.

> The size of the single file in a self-contained application will be large since it will include the runtime and the framework libraries.

https://docs.microsoft.com/de-de/dotnet/core/deploying/single-file

**Comment by yarseyah** [comment]
Indeed, whilst I hadn't looked yet at .net 5, in previous core versions - since you're bundling the whole of the "framework" too, this was the sort of sizes I was finding then.
